flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -744,6 +744,30 @@ public void testExecuteInsertInto() throws Exception {
 						fail("Unexpected job status.");
 				}
 			}
+			// Execute the same update again.
+			final ProgramTargetDescriptor targetDescriptor2 = executor.executeUpdate(
```

Comment:
looks like these are duplicated with previous codes, should avoid this

Reply:
Maybe we can write a for loop ?